tls/s2n_cipher_suites.c

```diff
@@ -1155,17 +1173,183 @@ int s2n_set_cipher_as_client(struct s2n_connection *conn, uint8_t wire[S2N_TLS_C
     return 0;
 }
 
-static int s2n_wire_ciphers_contain(const uint8_t *match, const uint8_t *wire, uint32_t count, uint32_t cipher_suite_len)
+/* This function returns the client index for a cipher *match. If a matching cipher could not be found, then -1 is returned. */
+static int s2n_wire_ciphers_has_server_cipher_at(const uint8_t *match, const uint8_t *wire, uint32_t count, uint32_t cipher_suite_len)
```

Comment:
Let's rename this to be more understandable.
```suggestion
static int s2n_wire_index_of_cipher_suite(const uint8_t *iana_value, const uint8_t *wire, uint32_t count, uint32_t cipher_suite_len)
```
Also formatting nit, can you add a tab when the line wraps?